### ISSUE TITLE
Hever: H2 heading is bigger than H1

### ISSUE THREAD

**Issue by AtrumGeost** [Automattic/themes#2329]
## Steps to replicate

1. Create a new page or post
2. Add a title
3. Add a Heading block

## Result

You'll notice the H2 is bigger than H1.

![Screen Shot 2020-08-05 at 16 21 42](https://user-images.githubusercontent.com/3696121/89465459-c23a5f80-d737-11ea-99fb-228fb7657d56.png)

## Expected

To have a visual hierarchy, where the H1 (title) text would be bigger

* User report: Dogfooding

**Comment by chad1008** [comment]
I tested this as well, and it looks like the visual hierarchy is present within the post, but the post/page titles are sized separately, outside of that hierarchy.

The end result is that H1s within the post content are 2.31306rem, which is slightly larger than the post title, while H2s within the content are the same size as the post title at 2.01136rem.

It's possible the smaller post title was an intentional design decision, but here's another screenshot with some labeled headings:

<img width="859" alt="Screen Shot 2020-08-17 at 10 14 12" src="https://user-images.githubusercontent.com/13856531/90405913-6eeada00-e072-11ea-80d1-5b6242f5a6f2.png">

I tested at a full width preview, but the sizing also appears consistent across smaller breakpoints as well.
